src/containers/Sidebar/ThreeDPane.tsx

```diff
@@ -130,6 +139,28 @@ class NodeDrawer extends React.Component<Props, State> {
       title = 'No Node Linked to Asset';
     } else if (rootAssetId && modelId && nodeId && assetId) {
       title = `Mapped Node: ${all[assetId] ? all[assetId].name : 'Loading...'}`;
+      form = (
+        <Popconfirm
+          title="Are you sure you want to unmap the 3D node?"
+          onConfirm={() => {
+            if (revisionId && modelId) {
+              this.props.deleteAssetNodeMapping(modelId, revisionId, asset!.id);
```

Comment:
An asset mapping is always an assetId-nodeId pair, no?

Reply:
Yes but it is also linked to model and revision as well (since nodeID is not globally unique)